Framework/MDAlgorithms/test/MaskMDTest.h

```diff
@@ -160,9 +161,9 @@ class MaskMDTest : public CxxTest::TestSuite {
     alg.initialize();
     alg.setPropertyValue("Workspace", wsName);
     alg.setPropertyValue("Dimensions", "Axis0, Axis1, Axis2");
-    alg.setPropertyValue("Extents", "0,10"); // wrong number of extents
-                                             // provided.
-    TS_ASSERT_THROWS(alg.execute(), std::invalid_argument);
+    alg.setPropertyValue("Extents", "0,10");  // wrong number of extents
+                                              // provided.
+    TS_ASSERT_THROWS_ANYTHING(alg.execute()); // fail input validators
```

Comment:
Instead of doing this. I would have called validateInputs directly. It's public on the Algorithm base class.